datanode/gateway/graphql/resolvers.go

```diff
@@ -1181,11 +1181,30 @@ func (r *myPartyResolver) TransfersConnection(
 	return r.r.transfersConnection(ctx, &party.Id, direction, pagination)
 }
 
-func (r *myPartyResolver) RewardsConnection(ctx context.Context, party *types.Party, assetID *string, pagination *v2.Pagination) (*v2.RewardsConnection, error) {
+func (r *myPartyResolver) RewardsConnection(ctx context.Context, party *types.Party, assetID *string, pagination *v2.Pagination, fromEpoch *int, toEpoch *int) (*v2.RewardsConnection, error) {
```

Comment:
Maybe if pagination would be the last parameter would be a bit cleaner, although I'd bundled these (party, assetID, fromEpoch, toEpoch) into a single filter struct. No biggie